app/src/main/java/ch/epfl/sdp/blindly/match/Profile.kt

```diff
@@ -37,7 +37,7 @@ data class Profile(
     /**
      * Describe the content as in Integer
      *
-     * @return content description
+     * @return content description (0)
```

Comment:
I didn't understand what this is doing only returning 0, maybe explain a bit more in the comments why this is needed.

Reply:
I agree

Reply:
return 1 if the object contains a content descriptor otherwise return 0